fr/index.html

```diff
@@ -0,0 +1,391 @@
+          >
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">Rendre un site web accessible est coûteux et prend du temps</h2>
+
+          <p>
+            Cela peut ne pas être le cas si l'accessibilité est prise en compte dès le début du projet et si l'équipe
+            produit dispose des compétences appropriées. Lorsque l'accessibilité est devenue une habitude,
+            <strong>le temps de développement ne change pas</strong> ou peu.
+          </p>
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">On peut s'occuper de l'accessibilité à la dernière minute</h2>
+
+          <p>
+            Cela ne marchera pas. Peut-être pour des choses simples comme ajouter les textes alternatifs ou les
+            attributs des formulaires, mais
+            <strong
+              >certaines experiences utilisateur complexes doivent être planifiées à l’avance pour être
+              accessibles</strong
+            >. Dans certains cas, rendre les fonctionnalités accessibles à la fin du processus de développement peut
+            nécessiter un remaniement complet.
+          </p>
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">L'accessibilité consiste uniquement à ajouter un texte alternatif aux images</h2>
+
+          <p>
+            Dans les faits, l'absence de texte alternatifs pour les images est l'<a
+              target="_blank"
+              href="https://webaim.org/projects/million/"
+              >un des principaux problèmes d'accessibilité qui soit détecté</a
+            >. Mais en dehors de cela, il y a beaucoup d'autres choses à prendre en compte lors de la création d'un site
+            web accessible - la structure des titres, l’interactivité des composants, le contraste des couleurs et bien
+            d'autres encore.
+          </p>
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">L’accessibilité du web est une responsabilité qui ne revient qu’aux développeurs</h2>
+          <p>
+            C'est un travail d'équipe. Les <strong>designers</strong> créent des interfaces utilisateur accessibles, les
+            <strong>développeurs</strong> les construisent, les <strong>ingénieurs AQ</strong> effectuent les tests
+            d'accessibilité, les responsables <strong>de projet</strong> s'assurent que l'accessibilité est incluse dans
+            les processus de l'équipe, l'<strong>équipe juridique</strong> vérifie qu'un produit ne présente aucun
+            risque du point de vue de l'accessibilité, les <strong>gestionnaires de contenu</strong> adaptent le contenu
+            pour qu'il soit conforme.
+          </p>
+
+          <a href="https://accessibility.digital.gov/" target="_blank"
+            >Plus de détails <span class="visually-hidden"></span
```

Comment:
@MarionCouesnon could you please add a myth name to `visually-hidden`.

Reply:
Done with 77c2a05d2459067877887858494997c996167cdd